### DISCUSSION TITLE
Errors after creating a new project

### DISCUSSION THREAD

**Discussion by GuiWolff** [unoplatform/uno#7826] [Q&A]
After creating a new UnoPlatform App | NetCore6.0, the project displays the following errors:

![image](https://user-images.githubusercontent.com/68097839/149573558-e37f3bfc-9968-4680-945a-f03804b9ee25.png)

How to fix?

**Comment by jeromelaban** [comment]
Thanks for the report! In general, this means that nuget did not restore the packages properly. You may take a look at the nuget output window for more information.
